package-system/AwsIotDeviceSdkCpp/build_config.json

```diff
@@ -89,7 +89,8 @@
                "-DCMAKE_OSX_DEPLOYMENT_TARGET=10.12",
```

Comment:
Mac is not officially supported by O3DE right now but I was able to build and test the Mac package with these build configurations. We can update it later when Mac becomes a platform we need to focus on.

Reply:
please update it, all uploaded packages should have correct osx version, or it will cause warning message on build as O3DE has the version set as 11.0
https://github.com/o3de/o3de/blob/5ae35810ae4d98f64b543fdd43cad6a092e2b04f/cmake/Platform/Mac/PAL_mac.cmake#L41